pkg/protocol/args.go

```diff
@@ -379,6 +379,17 @@ func (a *Args) PeekExists(key string) (string, bool) {
 	return peekArgStrExists(a.args, key)
 }
 
+// PeekMulti returns all the arg values for the given key.
+func (a *Args) PeekMulti(key string) [][]byte {
+	var values [][]byte
+	a.VisitAll(func(k, v []byte) {
+		if string(k) == key {
```

Comment:
If only used in line 386, `bytesconv.B2s(k)` may be a better choice - less allocation

Reply:
Thanks for your review, I checked B2s and found that it has updated its implementation in go1.20 
https://go.dev/src/strings/builder.go#L45
https://github.com/golang/go/issues/53003
https://github.com/valyala/fasthttp/blob/master/b2s_new.go

Reply:
About the updated version in go1.20, it is welcome to submit another PR to do so~